### ISSUE TITLE
Web site synchronization supports multiple root websites

### ISSUE THREAD

**Issue by zhanghx0905** [labring/FastGPT#805]
**例行检查**

- [x] 我已确认目前没有类似 features
- [x] 我已确认我已升级到最新版本
- [x] 我已完整查看过项目 README，已确定现有版本无法满足需求
- [x] 我理解并愿意跟进此 features，协助测试和提供反馈
- [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 features 可能会被无视或直接关闭**

**功能描述**

现在在线版知识库Web站点同步只允许填写一个网站的根地址，我希望能支持填写多个网站根地址，这样就不用每个根地址都新增一个知识库，使用起来容易很多。

**应用场景**

- [x] 在线版功能

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
**Routine inspection**

- [x] I have confirmed that there are currently no similar features
- [x] I have confirmed that I have upgraded to the latest version
- [x] I have fully reviewed the project README and determined that the existing version cannot meet the needs.
- [x] I understand and am willing to follow up on this feature, assist with testing and provide feedback
- [x] I understand and agree with the above content, and understand that project maintainers have limited energy. **Features that do not follow the rules may be ignored or closed directly**

**Function description**

Currently, the online version of knowledge base Web site synchronization only allows filling in the root address of one website. I hope to support filling in multiple website root addresses, so that there is no need to add a new knowledge base for each root address, and it is much easier to use.

**Application Scenario**

- [x] Online version features
